compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParserErrorHandler.java

```diff
@@ -4901,6 +4956,7 @@ private ParserRuleContext getNextRuleForVarName() {
         ParserRuleContext parentCtx = getParentContext();
         switch (parentCtx) {
             case ASSIGNMENT_STMT:
+            case GROUPING_KEY_LIST_VAR_DECLARATION:
```

Comment:
Do we need this change?

Reply:
Ya we need it when recovering scenarios like: `group by price = price1 + price2`